js/formidable_admin.js

```
@@ -2854,35 +2854,109 @@ function frmAdminBuildJS() {
 
-		fields = getFieldList();
-		list = document.getElementById( 'frm-calc-list-' + fieldId );
+		fields         = getFieldList();
+		list           = document.getElementById( 'frm-calc-list-' + fieldId );
 		list.innerHTML = '';
 
+		const supportsShowShortcodes = document.getElementById( 'new_fields' ).dataset.supportsShowShortcodes;
+
 		for ( i = 0; i < fields.length; i++ ) {
 			if ( ( exclude && exclude.includes( fields[ i ].fieldType ) ) ||
 				( excludedOpts.length && hasExcludedOption( fields[ i ], excludedOpts ) ) ) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)